### ISSUE TITLE
[Suggestion] Change the angled button to a switch

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#5751]
A very minor detail that's been bugging me for quite awhile. The current angled button is has an on/off display. wouldn't this make more sense as a switch? As a button, you only get 'on' for a second because it returns to 'off'.

![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/46c72a32-cfbe-4029-bc6d-bcb3d36a2a8d)

A replacement.
![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/7c6e21b0-3a3a-43e1-b324-1c17245485e7)

**Comment by tsunamayo** [comment]
@ZachZent Yeah right! The big problem is how do I proceed to not break everybody build? I could hide this block and make two new one (the proposed button, and the switch with the same look), but that would confuse player a bit.

**Comment by ZachZent** [comment]
Well in a case like this, wouldn't it be possible to update the in game model to a new angled button without changing any id values of the block itself? That way it would just swap without any issues. Sort of like a texture pack.


And since you mentioned the "double item" option, I'm going to take advantage of such by bringing up my old nemesis, the chair being too long. That double item option would work just as well. All it needs is it being shortened from 1m long to .75. Keep the player where they are and even in current builds, the only change would be 0.25x1x1m of empty space.

![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/79f22c88-6bb3-491b-a762-177440b11090)


Or for either, there is also the "this is early access and changes are to be expected" line.

**Comment by Majikmonster** [comment]
> @ZachZent Yeah right! The big problem is how do I proceed to not break everybody build? I could hide this block and make two new one (the proposed button, and the switch with the same look), but that would confuse player a bit.

The confusion would be a minor issue, as the flat button whose model the suggested change would be based on has a similar indication of on/off by its luminosity and position.

The new added brick would resemble the old one and function like a switch, which makes sense from a mechanical perspective as an interactive indicator. Or you could just change the button model and not add a second switch brick, instead adding an indicator light that uses the old sloped button model and on/off graphic instead of a color-changing light.
